src/adapters/networkHandlerFactory.js

```diff
@@ -16,16 +18,26 @@ SUPPORTED_VERSIONS.forEach((version) => {
   const destinations = getIntegrations(path.resolve(__dirname, `../${version}/destinations`));
   destinations.forEach((dest) => {
     try {
-      handlers[dest] = require(`../${version}/destinations/${dest}/networkHandler`).networkHandler;
+      // handles = {
+      //   v0: {
+      //     dest: handler
+      //   },
+      //   v1: {
+      //     dest: handler
+      //   },
+      // generic: GenericNetworkHandler,
+      // }
+      handlers[version][dest] =
+        require(`../${version}/destinations/${dest}/networkHandler`).networkHandler;
     } catch {
       // Do nothing as exception indicates
       // network handler is not defined for that destination
     }
   });
 });
 
-const getNetworkHandler = (type) => {
-  const NetworkHandler = handlers[type] || handlers.generic;
+const getNetworkHandler = (type, version) => {
```

Comment:
Can we add unit test for this  `src/adapters/networkHandlerFactory.js`

Reply:
done